### DISCUSSION TITLE
Overlays Meeting (2024-10-22)

### DISCUSSION THREAD

**Discussion by ralfhandl** [OAI/Overlay-Specification#85] [Meeting Notes]
## Agenda

Publication of Overlay 1.0.0

* #84 
* #86
* #87 

Once the above have been merged and the publish workflow has suceeded: 
* https://github.com/OAI/OpenAPI-Specification/pull/4145
* https://github.com/OAI/OpenAPI-Specification/pull/4144

**Comment by lornajane** [comment]
Discuss making post-release changes and what our process should be for those.

**Comment by lornajane** [comment]
During the meeting we published the 1.0.0 version of the specification to spec.openapis.org by merging #4145 and #4144
